libs/shared/components/src/auth/UserMenu/UserMenu.css.ts

```diff
@@ -19,6 +19,25 @@ export const dropdown = style({
   }),
 })
 
+export const fullScreen = style({
+  position: 'fixed',
+  top: spacing[4],
+  right: spacing[4],
+  maxHeight: `calc(100vh - ${spacing[6]}px)`,
```

Comment:
The modal now overflows in smaller windows, like small laptops and mobiles with some OS/browser chrome taking space from the website. The issue probably that there's now a fixed max-height for the delegation scroll-list while previously it flexed up and down to fill the screen vertically.

Before:

![image](https://user-images.githubusercontent.com/115094/146946687-eb215023-2c32-41e7-8114-55bf0dc8624a.png)

After:

![image](https://user-images.githubusercontent.com/115094/146946611-88b7af11-1dc0-423f-a816-a7df456c240e.png)

Reply:
This is a problem! I'll talk to Keli and find a solution on smaller screens

Reply:
Done ✅